addons/docs/package.json

```diff
@@ -131,6 +131,7 @@
     "require-from-string": "^2.0.2",
     "rxjs": "^6.6.3",
     "styled-components": "^5.2.1",
+    "sveltedoc-parser": "4.1.0",
```

Comment:
The newer version doesn't work correctly, some issue relating to a URL and eslint, strange. I made it an exact version in the devDependency, this way we have absolute control over the version WE use, but users can use whatever works for them.